tasks.py

```diff
@@ -12,21 +12,98 @@
 # "PYTHONUNBUFFERED" env var for Windows at True.
 if platform == 'win32':
-    PLATFORM_ARG = dict(env={'PYTHONUNBUFFERED': 'True', 'PIPENV_DONT_LOAD_ENV': '1'})
+    PLATFORM_ARG = dict(env={'PYTHONUNBUFFERED': 'True'})
 else:
-    PLATFORM_ARG = dict(pty=True, env={'PIPENV_DONT_LOAD_ENV': '1'})
+    PLATFORM_ARG = dict(pty=True)
+
+
+def create_env_file(env_file):
+    """Create or update the .env file"""
+    with open(env_file, 'r') as f:
+        env_vars = f.read()
+    # update the DATABASE_URL env
+    new_db_url = "DATABASE_URL=postgres://postgres@localhost:5432/pulse"
+    old_db_url = re.search('DATABASE_URL=.*', env_vars)
+    if old_db_url:
+        env_vars = env_vars.replace(old_db_url.group(0), new_db_url)
+    else:
+        env_vars = env_vars + "DATABASE_URL=postgres://postgres@localhost:5432/pulse\n"
+
+    # update the ALLOWED_HOSTS env
+    new_hosts = "ALLOWED_HOSTS=*"
+    old_hosts = re.search('ALLOWED_HOSTS=.*', env_vars)
+    if old_hosts:
+        env_vars = env_vars.replace(old_hosts.group(0), new_hosts)
+    else:
+        env_vars = env_vars + "ALLOWED_HOSTS=*\n"
+
+    # create the new env file
+    with open('.env', 'w') as f:
+        f.write(env_vars)
+
+
+def create_super_user(ctx):
+    preamble = "from django.contrib.auth import get_user_model;User = get_user_model();"
+    create = "User.objects.create_superuser('admin', 'admin@mozillafoundation.org', 'admin')"
+    manage(ctx, f'shell -c "{preamble} {create}"')
+    print("\nCreated superuser `admin@mozillafoundation.org` with password `admin`.")
+
+
+# Project setup and update
+@task(aliases=["new-env"])
+def setup(ctx):
+    """Automate project's configuration and dependencies installation"""
+    with ctx.cd(ROOT):
+        if os.path.isfile(".env"):
+            print("* Updating your .env")
+            create_env_file(".env")
+        else:
+            print("* Creating a new .env")
+            create_env_file("sample.env")
+        # create virtualenv
+        if not os.path.isfile("./pulsevenv/bin/python"):
+            print("* Creating a Python virtual environment")
+            ctx.run("python3 -m venv pulsevenv")
+            print("* Installing pip-tools")
+            ctx.run("./pulsevenv/bin/pip install pip-tools")
+        # install deps
+        print("* Installing Python dependencies")
+        pip_sync(ctx)
+        new_db(ctx)
+
+
+@task(aliases=["catchup"])
+def catch_up(ctx):
+    """Install dependencies and apply migrations"""
+    print("Installing Python dependencies")
+    pip_sync(ctx)
+    print("Applying database migrations")
+    migrate(ctx)
 
 
-@task(optional=['option', 'flag'])
-def manage(ctx, command, option=None, flag=None):
-    """Shorthand to manage.py. inv manage [COMMAND] [-o OPTION] [-f FLAG]. ex: inv manage runserver -o 3000"""
+@task
+def new_db(ctx):
+    """Create a new database with fake data"""
+    print("* Reset the database")
+    ctx.run("dropdb --if-exists pulse")
```

Comment:
This command fails to connect to a database.

```
(pulsevenv)  cade@X1MOZCARBON  ~/code/network-pulse-api   remove-pipenv  inv new-db                                               
* Reset the database
dropdb --if-exists pulse
dropdb: could not connect to database template1: could not connect to server: No such file or directory
        Is the server running locally and accepting
        connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
```

Reply:
started the postgres container with `docker run -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432  postgres`